clair/values.yaml

```diff
@@ -0,0 +1,79 @@
+# Default values for clair.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+logLevel: info
+insecureTls: false
+image:
+  repository: quay.io/coreos/clair-git
+  tag: latest
+  pullPolicy: Always
+service:
+  type: NodePort
+  internalApiPort: 6060
```

Comment:
In our cluster is *almost* impossible to have different internal and external ports, therefore you can just keep one port each, like: `apiPort`, `healthPort`